src/Stratis.Bitcoin.IntegrationTests/Mempool/MempoolRelaySpecification_Steps.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System.Linq;
+using FluentAssertions;
+using FluentAssertions.Common;
+using NBitcoin;
+using Stratis.Bitcoin.Connection;
+using Stratis.Bitcoin.Features.Consensus;
+using Stratis.Bitcoin.IntegrationTests.EnvironmentMockUpHelpers;
+using Xunit.Abstractions;
+
+namespace Stratis.Bitcoin.IntegrationTests.Mempool
+{
+    public partial class MempoolRelaySpecification
+    {
+        private NodeBuilder nodeBuilder;
+        private CoreNode nodeA;
+        private CoreNode nodeB;
+        private CoreNode nodeC;
+        private int coinbaseMaturity;
+        private Transaction transaction;
+
+        // NOTE: This constructor is allows test steps names to be logged
+        public MempoolRelaySpecification(ITestOutputHelper outputHelper) : base(outputHelper)
```

Comment:
I am wondering whether we just enforce this on the base BddSpecification and not even have the default constructor - Logging the times will almost always be useful to have.

Reply:
I'll let you deal with that later